### PR TITLE
added: ListItem.setProperties wrapper for python list items

### PR DESCRIPTION
**Pull request by notspiff**
successively call setProperty based on a passed dictionary

ref https://forum.kodi.tv/showthread.php?tid=332283

### CODE REVIEW

File: xbmc/interfaces/legacy/ListItem.h [hunk 1]
```diff
@@ -944,6 +944,28 @@ namespace XBMCAddon
...
+      /// @param values             dictionary - pairs of `{ label: value }`.
+      ///-----------------------------------------------------------------------
+      /// **Example:**
+      /// ~~~~~~~~~~~~~{.py}
+      /// ...
+      /// # setProperties(values)
+      /// listitem.setProperties({ 'AspectRatio': '1.85', 'StartOffset' : '256.4' })
+      /// ...
+      /// ~~~~~~~~~~~~~
+      ///
```
Comment (Rechi) [inline xbmc/interfaces/legacy/ListItem.h]: `@python_v18 New function added.`

Comment (notspiff) [reply xbmc/interfaces/legacy/ListItem.h]: rebased.

**Comment by notspiff** [comment]
jenkins build this please

**Comment by Rechi** [comment]
@notspiff ready for merging?

**Comment by notspiff** [comment]
yes.

**Pull request merged**